src/person/actions/delete-address.action.ts

```diff
@@ -17,8 +18,20 @@ export const deleteAddress = async (
         throw errorFactory.create(PersonErr.DELETE_ADDRESS_FAILED);
       });
 
-    return response.data;
+    switch (response.status) {
+      case 200:
+        return response.data;
+      case 403:
+        throw errorFactory.create(PersonErr.DELETE_ADDRESS_FORBIDDEN);
+      case 404:
+        throw errorFactory.create(PersonErr.DELETE_ADDRESS_NOT_FOUND);
```

Comment:
Pour les erreurs 404 c'est préférable (du moins pour le moment) d'avoir des codes d'erreur qui font uniquement référence à la ressource, et non pas un code différent pour chaque route. On devrait donc avoir : 

```suggestion
        throw errorFactory.create(PersonErr.ADDRESS_NOT_FOUND);
```